bindings/wasm/bindings.cpp

```diff
@@ -55,6 +56,48 @@ std::vector<SimplePolygon> ToPolygon(
   return simplePolygons;
 }
 
+val GetMeshJS(const Manifold& manifold) {
+  MeshGL mesh = manifold.GetMeshGL();
+  val meshJS = val::object();
+
+  meshJS.set("triVerts",
+             val(typed_memory_view(mesh.triVerts.size(), mesh.triVerts.data()))
+                 .call<val>("slice"));
```

Comment:
Yeah, some of their docs are pretty well-hidden: https://emscripten.org/docs/porting/connecting_cpp_and_javascript/embind.html?highlight=typed_memory_view#memory-views and `slice` is just the JS Array method; the cool thing is [`val.call`](https://emscripten.org/docs/api_reference/val.h.html#_CPPv4NK10emscripten10emscripten3val4callEPKcDpRR4Args) lets you call any JS function you want.

Now that I know about this, I'm thinking it would be better to move more of our logic from `bindings.js` to `bindings.cpp` so we can minimize the footprint of our exposed bindings.

Reply:
Just a comment. I'm not sure if I'm right, but I feel there is some overhead in using the `val` as it has to switch context between JS and wasm. It might be better to use [`EM_JS`](https://emscripten.org/docs/api_reference/emscripten.h.html#c.EM_JS) to do all the javascript work in one context switch.

Reply:
That may be true, but `val` is also vastly easier to use (no restrictions on types). Still, that may be reason enough not to bother moving the other logic. Some perf comparisons would be interesting!